admin_manual/enterprise/external_storage/windows-network-drive_configuration.rst

```diff
@@ -154,6 +154,121 @@ In openSUSE, modify the ``/usr/sbin/start_apache2`` file::
 
 Restart Apache, open your ownCloud Admin page and start creating SMB/CIFS mounts.
 
+
+Create Shares programmatically
+------------------------------
+
+Some users want to create new shares by command line scripts. There are two different approaches.
+
+### 1. occ files_external:create
+
+
+**One liner - personal share**
+
+::
+    occ files_external:create /WND windows_network_drive password::logincredentials --config={host=127.0.0.1,share='home',root='$user',domain='owncloud.local'} --user someuser
+
+**One liner - general share**
+
+::
+    occ files_external:create /WND windows_network_drive password::logincredentials --config={host=127.0.0.1,share='home',root='$user',domain='owncloud.local'}
+
+ 
+Arguments
+~~~~~~~~~
+
+> Usage::
+
+
+    files_external:create [options] [--] <mount_point> <storage_backend> <authentication_backend>
+
+::
+
+    Argument | Options
+    -------------|-----------
+    mount point | Path of the mount point within the file system
+    storage_backend | Storage backend indentifier
+    authentication_backend | authentication backend authentifier
+
+    Storage Backend | Identifier
+    ------------------------|-------------
+    Windows Network Drive | windows_network_drive
```

Comment:
You have ``Windows Network Drive | windows_network_drive``
but write above: ``occ files_external:create /WND windows_network_drive``
how does that relate

Reply:
From my limited knowledge it means the storage backend, that's also the name of the column. 

the command is `occ`, so far so good, then `files_external:create`, this is also plausible, right? then the name of the share `/wnd` and then the storage backend - `windows_network_drive`

the syntax for the command is above:
```
Arguments

Usage: files_external:create [options] [--] <mount_point> <storage_backend> <authentication_backend>
````

Reply:
> the name of the share /wnd

proposal to avoid confusion: ``occ files_external:create /my_share_name windows_network_drive``

Reply:
sure